OpenUpSA/wazimap-ng-ui
# adding back the link to Google maps in the facility modal

## Description
adding a view location in Google Maps link in the facility modal.

Merged https://github.com/OpenUpSA/wazimap-ng-ui/pull/474 into this to have the cypress related changes

## Related Issue
https://trello.com/c/exSNZzO7/887-point-details-google-map-button-disappeared-again

## How to test it locally
* on the point mapper click on any category
* wait until the point markers are created on the map
* click on a point marker
* on the tooltip click on Show More button to view the facility modal
* confirm that in the facility modal, we now have the Google Maps link and it redirects to the correct coordinates

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/129634432-e855ec96-71f6-49fa-83d1-eeca7f4daf83.png)


## Changelog

### Added

### Updated
* `index.html` and wazimap-ng-v1.webflow.css to apply design changes
* `point_data.js` to modifiy href property of the link button

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
